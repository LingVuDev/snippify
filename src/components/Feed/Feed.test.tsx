import { render } from "@testing-library/react";
import Feed from ".";

describe('Feed', () => {

  it('renders', () => {
    const { container } = render(<Feed />);

    expect(container).toMatchSnapshot();
  })
});
