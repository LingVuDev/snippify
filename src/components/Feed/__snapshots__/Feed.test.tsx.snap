// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed renders 1`] = `
<div>
  <div>
    Feed
  </div>
</div>
`;
